src/blockchain_db/lmdb/db_lmdb.cpp

```diff
@@ -1516,21 +1516,55 @@ void BlockchainLMDB::update_txpool_tx(const crypto::hash &txid, const txpool_tx_
   }
 }
 
-uint64_t BlockchainLMDB::get_txpool_tx_count() const
+uint64_t BlockchainLMDB::get_txpool_tx_count(bool include_unrelayed_txes) const
 {
   LOG_PRINT_L3("BlockchainLMDB::" << __func__);
   check_open();
 
-  TXN_PREFIX_RDONLY();
   int result;
+  int num_entries = 0;
```

Comment:
Same type as the return value to avoid implicit conversions (that said, this didn't really need to return uint64_t in the first place, but since it does...)

Reply:
ok